pkgs/test/README.md

```diff
@@ -233,6 +234,25 @@ dart test --total-shards 3 --shard-index 0 path/to/test.dart
 dart test --total-shards 3 --shard-index 1 path/to/test.dart
 dart test --total-shards 3 --shard-index 2 path/to/test.dart
 ```
+Sharding: This refers to the process of splitting up a large test suite into
+smaller subsets (called shards) that can be run independently. Sharding is
+particularly useful for distributed testing, where multiple machines are used
+to run tests simultaneously. By dividing the test suite into smaller subsets,
+you can run tests in parallel across multiple machines, which can significantly
+reduce the overall testing time.
+
+### Concurrency Testing
+
+Tests run concurrently by default on two threads. use `--concurrency` to control
```

Comment:
I believe the default is actually to use half of the cores available on your computer

Reply:
From dart test --help:

> -j, --concurrency=<threads>           The number of concurrent test suites run.
>                                       (defaults to "2")

But I do not have a way to confirm that.

Reply:
https://github.com/dart-lang/test/blob/e56c643950fb06a9d249241715730561447ed24f/pkgs/test/test/runner/runner_test.dart#L51 that message is actually specific to your computer

Reply:
That makes sense I will fix that, thanks

Reply:
Oh sorry that is just the test, here is the actual default https://github.com/dart-lang/test/blob/e56c643950fb06a9d249241715730561447ed24f/pkgs/test_core/lib/src/runner/configuration/values.dart#L14